website/_docs30/install/kylin_docker.md

```diff
@@ -1,10 +1,143 @@
+KYLIN_HOME=/home/admin/apache-kylin-3.0.0-alpha2-bin-hbase1x
+{% endhighlight %}
+
+## Build image to verify source code modifications
+
+The docker image can also be used when developers have modified the source code and want to package, deploy, and verify the source code. First, we go to the docker directory under the root directory of the source and execute the script below to build the image and copy the source into the image.: 
+
+{% highlight Groff markup %}
+#!/usr/bin/env bash
+
+echo "start build kylin image base on current source code"
+
+rm -rf ./kylin
+mkdir -p ./kylin
+
+echo "start copy kylin source code"
+
+for file in `ls ../../kylin/`
+do
+    if [ docker != $file ]
+    then
+        cp -r ../../kylin/$file ./kylin/
+    fi
+done
+
+echo "finish copy kylin source code"
+
+docker build -t apache-kylin-standalone .⏎
+{% endhighlight %}
+
+Due to need to download and deploy various installation packages over the network, the entire build process can last for tens of minutes, depending on the network.
+
+When the image build is complete, execute the following command to start the container: 
+
+{% highlight Groff markup %}
+docker run -d \
+-m 8G \
+-p 7070:7070 \
+-p 8088:8088 \
+-p 50070:50070 \
+-p 8032:8032 \
+-p 8042:8042 \
+-p 60010:60010 \
+apache-kylin-standalone
+{% endhighlight %}
+
+When the container starts, execute the docker exec command to enter the container. The source code is stored in the container dir `/home/admin/kylin_sourcecode`, execute the following command to package the source code: 
+
+{% highlight Groff markup %}
+cd /home/admin/kylin_sourcecod
+build/script/package.sh
+{% endhighlight %}
+
+After the package is complete, an installation package ending in `.tar.gz` will be generated in the `/home/admin/kylin_sourcecode/dist` directory, such as `apache-kylin-3.0.0-alpha2-bin-hbase1x.tar.gz`. We can use this installation package to deploy and launch Kylin services such as:
+
+{% highlight Groff markup %}
+cp /home/admin/kylin_sourcecode/dist/apache-kylin-3.0.0-alpha2-bin-hbase1x.tar.gz /home/admin
+tar -zxvf /home/admin/apache-kylin-3.0.0-alpha2-bin-hbase1x.tar.gz
+/home/admin/apache-kylin-3.0.0-alpha2-bin-hbase1x/kylin.sh start
+{% endhighlight %}
+
+We can also open pages for services such as Hdfs, Yarn, HBase, and Kylin in the browser of this machine.
+
+## Container resource recommendation
+
+In order to allow Kylin to build the cube smoothly, the memory resource we configured for Yarn NodeManager is 6G, plus the memory occupied by each service, please ensure that the memory of the container is not less than 8G, so as to avoid errors due to insufficient memory.
+
+For the resource setting method for the container, please refer to:
+
+- Mac user: <https://docs.docker.com/docker-for-mac/#advanced>
+- Linux user: <https://docs.docker.com/config/containers/resource_constraints/#memory>
+
+---
+
+Old docker image please check the github project [kylin-docker](https://github.com/Kyligence/kylin-docker/).
```

Comment:
As far as I see, I don't think this sentence is not fluent in terms of English grammar.

Maybe "For old docker image, please check the github page ...." will be better.